Root.tsx

```diff
@@ -19,19 +18,36 @@ import LoginScreen from "./screens/login/LoginScreen";
 import * as SecureStore from "expo-secure-store";
 import { setUserToken } from "./helpers/axios/axiosConfig";
 import styles from "./RootStyle";
+import UpdateInfoScreen from "./screens/updatePatientInfo/UpdateInfoScreen";
+import { Roles } from "./constants/roles/Roles";
 
-const HomeStack = createNativeStackNavigator<HomeStackParamList>();
 const AuthStack = createNativeStackNavigator<AuthStackParamList>();
 const Drawer = createDrawerNavigator<DrawerStackParamList>();
 
-const HomeStackNavigator = () => {
+const DrawerStackNavigator = () => {
   return (
-    <HomeStack.Navigator screenOptions={{ headerShown: false }}>
-      <HomeStack.Screen name="Home" component={HomeScreen} />
-    </HomeStack.Navigator>
+    <Drawer.Navigator
+      initialRouteName="Home"
+      drawerContent={(props) => <DrawerContent {...props} />}
+      screenOptions={{
+        drawerActiveBackgroundColor: "black",
+        drawerActiveTintColor: "white",
+        headerShown: true,
```

Comment:
I think it's not necessary to specify `headerShown: true`, since its default value is `true`.